specification/cognitiveservices/data-plane/Speech/SpeechToText/preview/v3.2-preview.1/speechtotext.json

```diff
@@ -5042,12 +5235,92 @@
           "type": "string",
           "readOnly": true
         }
-      }
+      },
+      "readOnly": true
     },
     "EvaluationProperties": {
       "title": "EvaluationProperties",
       "type": "object",
       "properties": {
+        "tokenErrorRate2": {
```

Comment:
The numerical suffix on the property name is a bit odd. Should we group them in an array of objects? Or make each value an array and use the index of the item as the model index instead of embedding it in the name...

Reply:
We fully agree that the naming is not good and we will replace this flat list in next major version.
For 3.2-preview.1, we want to extend the schema used in 3.0 and 3.1 (with index in name) in order to be backward compatible (only adding new properties with similar naming).